### PR TITLE
ISWAP default mixed native support

### PR DESCRIPTION
**Pull request by isaacdevlugt**
**Description of the Change:** Native support for `qml.ISWAP` on `default.mixed`

**Benefits:** Native support

- [x] Make changelog entry

### CODE REVIEW

**Comment by antalszava** [comment]
Hi @isaacdevlugt, this is looking good, could we have a small test case too? We have some already for `default.qubit`: https://github.com/PennyLaneAI/pennylane/blob/master/tests/devices/test_default_qubit.py#L223

**Comment by isaacdevlugt** [comment]
Hey @antalszava! Made some tests 😄

File: tests/devices/temp.ipynb [hunk 1]
```diff
@@ -0,0 +1,159 @@
+{
```
Comment (antalszava) [inline tests/devices/temp.ipynb:1]: We should have this file removed from the PR and/or having a test case in the suite identical to what's being tested here.

**Review by antalszava — APPROVED** [review]
Looks good :+1: Approval is conditional on not having the iPython notebook file added as we have our test suite dedicated to tests.

File: tests/devices/temp.ipynb (hunk 1 above)
Comment (isaacdevlugt) [reply tests/devices/temp.ipynb:1]: Oops sorry! Total accident

**Pull request merged**